src/settings/lib/color-palette.scss

```diff
@@ -1,3 +1,23 @@
-$tbds-brand-red: #e03131 !default;
-$tbds-brand-gray: #29292c !default;
-$tbds-brand-blue: #0b758c !default;
+$tbds-color-brand-red: #e03131 !default;
+$tbds-color-brand-gray: #29292c !default;
+$tbds-color-brand-blue: #0b758c !default;
+
+$tbds-default-gray: #d4e1e4 !default;
```

Comment:
Should this block of colors adopt the prefix `tbds-color-…` ?

Also, I'm unsure about the name `default-gray`, perhaps the default alert should use one of the gray values?

Reply:
I agree with both points here.